packages/keycodes/README.md

```diff
@@ -0,0 +1,33 @@
+# @wordpress/keycodes
+
+Keycodes utilities for WordPress, used to check the key pressed in events like `onKeyDown`. Contains keycodes constants for keyboard keys like `DOWN`, `UP`, `ENTER`, etc.
+
+Example usage:
+
+```js
+import { DOWN, ENTER } from '@wordpress/keycodes';
+
+// [...]
+
+onKeyDown( event ) {
+	const { keyCode } = event;
+	
+	if ( keyCode === DOWN ) {
+		alert( 'You pressed the down arrow!' );
+	} else if ( keyCode === ENTER ) {
+		alert( 'You pressed the enter key!' );
+	} else {
+		alert( 'You pressed another key.' );
+	}
+}
+```
+
+## Installation
```

Comment:
I think I'd prefer if we keep the "installation" section before the "usage". It's fine to discuss these but it's important to keep our packages consistent.

Reply:
Totally right, good call. I moved it under "installation"

Reply:
Thanks for the change 👍